example/Makefile

```diff
@@ -0,0 +1,10 @@
+COQMAKEFILE ?= Makefile.coq
+
+include coqdocjs/Makefile.doc
```

Comment:
Your example makes `html` the default target. Is this intended? More generally, it might be useful to take as example `Makefile` something that also includes all the targets from the Coq Makefile and defaults to `all` in `Makefile.coq`.